ACCEPTANCE_TESTING.md

```
@@ -0,0 +1,171 @@
+   ```bash
+   just test-pr
+   ```
+3. **Select a PR** from the interactive menu by entering its number
+4. **Optionally clear the PostgreSQL volume** if you want a fresh database
+5. Done! The application will start automatically
+
+## Access the Application
+
+Once running, open your browser:
+
+- **Main UI**: http://localhost:8501
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)